File: PHP PowerPoint Examples/GeneralApiDemo/convertInRequestToPath.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\GeneralApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new GeneralApi($configuration);

$format = "Pdf"; //Available format: Pptx, Ppt, Odp, Pps, Ppsx, Pdf, Xps, Ps, Pcl,
$outPath = "output/ConvertInRequestToPath.pdf";
$password = null;
$filename = "D:/data/ConvertInRequestToPath.pptx";
$document = new SplFileObject($filename);;
$result = $apiInstance->convertInRequestToPath($format, $outPath, $document, $password);
print_r($result);

File: Python PowerPoint Examples/GeneralApiDemo/convertInRequestToPath.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.general_api.GeneralApi(configuration)

format = "Pptx"  # Supported formats:Pptx/Ppsx/Ppt/Pps/Odp/Pdf/Xps/Ps/Pcl
destFilePath = "output.pptx"
file = "D:\\input\\Convert.pptx"
password = ""

result = api.convert_in_request_to_path(format=format, dest_file_path=destFilePath, file=file, password=password)
